### DISCUSSION TITLE
www.Condorcet.vote

### DISCUSSION THREAD

**Discussion by julien-boudry** [julien-boudry/Condorcet#104] [Your Condorcet Projects]
[Condorcet.vote](https://www.condorcet.vote/) is a very simple technology demonstration for  Condorcet PHP as a backend to a voting service with a web interface.

It's not finished (and will never be) and not well maintained (not planned to do better). But it works for simple cases!

**Comment by LiamM32** [comment]
Would this work if I want to make an online poll with the Schulze method? If so, is there any mechanism to make it harder for people to vote twice (such as either a log-in or IP address), or would it have to be self-enforced by honour?

I do have an idea for a public vote that may be rather popular.

**Reply by julien-boudry** [reply]
Hi @LiamM32 ,

You can check the votes list. And also delete votes.

You can use the dedicated link :
![image](https://github.com/julien-boudry/Condorcet/assets/4020317/796226e6-b0cf-49a2-bc06-d3c4ec1a6c69)

**Reply by LiamM32** [reply]
How much interest would you have in making a fancier version of this? A way to make public polls that are counted with Condorcet, but with a more intuitive user interface for entering votes?

I think a drag-&-drop interface would be ideal. But I don't know how difficult this would be to make.

**Reply by julien-boudry** [reply]
This one is just an old unfinished demo, mainly coded in two weeks.

I'm working on another project around condorcet.vote. Not ready before 2024.

**Comment by LiamM32** [comment]
I'm interested in making a fork of this to try to improve the user interface. But I've never done web development before.

The first thing I must learn how to do is to run the website locally. But if I try to run it from Netbeans using the default settings as a local web site (which runs `index.php`), it just opens a Firefox window at `https://localhost/Condorcet.Vote/index.php` & says "Unable to connect."

**Comment by julien-boudry** [comment]
Hi @LiamM32 ,

I'm not guaranteed at all any updates to the actual prod. I may have other projects for this brand.

Good luck with the actual code, he is smart, but really exotic non-standard, and unfinished.

You can have a complete env. (web server, database) with Docker:
https://github.com/julien-boudry/Condorcet.Vote/blob/master/prod.md
